apps/react-world/src/components/home/HomeFeedTabItem.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
+
+interface HomeFeedTabItemProps {
+  isActive: boolean;
+  href?: string;
+  children: React.ReactNode;
+}
+
+const HomeFeedTabItem = styled.a<HomeFeedTabItemProps>`
+  display: block;
+  padding: 0.5em 1em;
+  border-radius: 0;
+  border: none;
+  border-bottom: 2px solid transparent;
+  background: transparent;
+  color: #aaa;
+  text-decoration: none; // 기본 링크 스타일 제거
+  cursor: pointer;
+
+  &:hover {
+    color: #555;
+    text-decoration: none; // hover 상태에서의 링크 스타일도 제거
+  }
+
+  ${props =>
+    props.isActive &&
+    `
+    background: #fff;
+    border-bottom: 2px solid #5cb85c;
+    color: #5cb85c;
+
+    &:hover {
+      color: #5cb85c; // isActive가 true일 때 hover 색상을 유지
+    }
+  `}
```

Comment:
```suggestion
${({ isActive }) => isActive && `
  background: #fff;
  border-bottom: 2px solid #5cb85c;
  color: #5cb85c;
  &:hover {
    color: #5cb85c; // isActive가 true일 때 hover 색상을 유지
  }
`}
```

구조분해 할당을 적용한다면 조금 더 간결하게 작성할 수 있을 것 같아 메시지 남겨보아요~!

Reply:
@hhhminme 오 코멘트 감사합니다! 반영했어요 ㅎㅎ